release-notes/VERSION-2.x

```diff
@@ -13,16 +13,12 @@ Project: jackson-databind
 #4788: `EnumFeature.WRITE_ENUMS_TO_LOWERCASE` overrides `@JsonProperty` values
  (reported by Mike M)
  (fix by Joo-Hyuk K)
+#4790: Fix `@JsonAnySetter` issue with "setter" method (related to #4639)
+ (reported by @bsa01)
+ (fix by Joo-Hyuk K)
 
 2.18.1 (28-Oct-2024)
 
-#4741: When `Include.NON_DEFAULT` setting is used on POJO, empty values
```

Comment:
Fixed ordering, added missing entry; not related to changes here but was convenient to fix. :)